src/etc/roles/literal/markRole.js

```diff
@@ -9,12 +9,12 @@ const markRole: ARIARoleDefinition = {
   nameFrom: [
     'prohibited',
   ],
-  prohibitedProps: [],
-  props: {
-    'aria-braillelabel': null,
-    'aria-brailleroledescription': null,
-    'aria-description': null,
-  },
+  prohibitedProps: [
+    'aria-braillelabel',
+    'aria-label',
+    'aria-labelledby',
+  ],
+  props: {},
```

Comment:
this part seems like a breaking change

Reply:
This is a bugfix (accidentally missed off the changelog), the `mark` role has always had the listed props as prohibited.

REF: https://www.w3.org/TR/wai-aria-1.3/#mark

Worth noting that the `mark` role is also new as of WAI-ARIA 1.3 so probably should not have been added to this package given it is currently tracking WAI-ARIA 1.2.